app/views/projects/_form.html.erb

```diff
@@ -2,8 +2,42 @@
 
   <%= submit_button(form: f, button: button, center: true) %>
 
-  <%= check_box_with_label(form: f, field: :accepting_observations,
-                           label: :form_projects_accepting_observations.t) %>
+  <%= tag.div(
+    tag.p("*#{:form_projects_date_range.t}*".t) +
+    tag.p(:form_projects_dates_explain.t)
+  ) %>
+  <div class="container-text ml-4">
+    <div>
+      <%= f.label(:start_date, "#{:form_projects_start_date.t}: ") %>
+      <%= f.date_select(:start_date,
+                        { class: "form-control mb-2" }) %>
+      <p>
+        <%= radio_button(
+              "start_date", "fixed", true, checked: @start_date_fixed
+            ) %> <%= :form_projects_date_fixed.t %>
+      </p>
+      <p>
+        <%= radio_button(
+            "start_date", "fixed", false, checked: !@start_date_fixed
+            ) %> <%= :form_projects_date_indefinite.t %>
+      </p>
+    </div>
+    <div>
+      <%= f.label(:end_date, "#{:form_projects_end_date.t}: ") %>
+      <%= f.date_select(:end_date,
+                        { class: "form-control mb-2" }) %>
+      <p>
+        <%= radio_button(
+              "end_date", "fixed", true, checked: @end_date_fixed
+            ) %> <%= :form_projects_date_fixed.t %>
+      </p>
+      <p>
+        <%= radio_button(
+              "end_date", "fixed", false, checked: !@end_date_fixed
+            ) %> <%= :form_projects_date_indefinite.t %>
+      </p>
```

Comment:
@mc-nathan, @nimmolo 
Not a big deal. But is there a way to set up these checkboxes so that the params are:
1. nested under [:project] (rather than being top-level); and
2. names something like `fixed_start` and `fixed-end` with values T/F (or 1/0).

Reply:
I just looked at what iNat does (Andy Wilson gave me admin privileges on the most recent NAMA project).  They have one set of radio buttons which allow you to choose between "Any", "Exact" (single data), "Range" (with calendar selector for start and end date), and "Months".  I like the simplicity of "Any" and "Range" being the options rather the concept of "Indefinite".  I can't really think of a reason why I'd want a project that had just a start date or just an end date.  I don't think we have need for the "Exact" or "Months" options.  In this context, I would think if "Any" is chosen, then no date stuff is show on the Project#show page.  If "Range" is chosen, then I'd put the range on top of the project image like the location info.

Reply:
Thanks. 
I like the simplicity of just Any exor Range.
Projects with just a start date or just an end date seem like an edge case, so I won't worry about them.
I'll think through how to do this. 
And It would be good to have UI that's understood by iNat project admins.
I think an actual calendar might require js, so I might go with date pickers for now.